central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/JBossCentralEditor.java

```diff
@@ -197,13 +199,15 @@ protected void addPages() {
 			}
 			setPageImage(index, gettingStartedImage);
 
-			softwarePage = new SoftwarePage(this);
-			index = addPage(softwarePage);
-			if (softwareImage == null) {
-				softwareImage = JBossCentralActivator.getImageDescriptor(
-						"/icons/software.png").createImage();
+			if (!new EclipseUserAgent().getApplicationName().contains("rh-eclipse")) { // Most likely RPM
```

Comment:
Can you pop an alert right before the if check so that the user (me) can see what my EclipseUserAgent().getApplicationName() contains? Or is "pop an alert" only something we can do in javascript debugging? :D